client/print-test-label/index.js

```diff
@@ -25,6 +25,10 @@ export default ( { paperSize, storeOptions } ) => ( {
 		};
 	},
 
+	getStateKey() {
```

Comment:
It's used in `client/main.js`, to persist the Redux state in storage if the user is forced to do a reload. Not really needed for something as simple as the "Print test label" form, but it's required or the app 💥